web_pwa_oca/templates/assets.xml

```diff
@@ -2,23 +2,27 @@
 <odoo>
     <template id="web_layout_pwa" name="Web layout PWA" inherit_id="web.layout">
         <xpath expr="//meta[@name='viewport']" position="after">
-            <!-- Add link rel manifest -->
-            <link rel="manifest" t-attf-href="/web_pwa_oca/manifest.webmanifest"/>
-            <!-- Add iOS meta tags and icons -->
-            <meta name="apple-mobile-web-app-capable" content="yes"/>
-            <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
-            <t t-set="pwa_name" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.name')"/>
-            <meta name="apple-mobile-web-app-title" t-att-content="pwa_name"/>
-            <link rel="apple-touch-icon" href="/web_pwa_oca/static/img/icons/icon-152x152.png"/>
-            <!-- Add meta theme-color -->
-            <t t-set="pwa_theme_color" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.theme_color')"/>
-            <meta name="theme-color" t-att-content="pwa_theme_color" />
+            <t t-if="not website">
+                <!-- Add link rel manifest -->
+                <link rel="manifest" t-attf-href="/web_pwa_oca/manifest.webmanifest"/>
+                <!-- Add iOS meta tags and icons -->
+                <meta name="apple-mobile-web-app-capable" content="yes"/>
+                <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
+                <t t-set="pwa_name" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.name')"/>
+                <meta name="apple-mobile-web-app-title" t-att-content="pwa_name"/>
+                <link rel="apple-touch-icon" href="/web_pwa_oca/static/img/icons/icon-152x152.png"/>
+                <!-- Add meta theme-color -->
+                <t t-set="pwa_theme_color" t-value="request.env['ir.config_parameter'].sudo().get_param('pwa.manifest.theme_color')"/>
+                <meta name="theme-color" t-att-content="pwa_theme_color" />
+            </t>
         </xpath>
     </template>
     <template id="assets_backend" name="web service worker assets" inherit_id="web.assets_backend">
         <xpath expr="." position="inside">
-            <script type="text/javascript" src="/web_pwa_oca/static/src/js/pwa_manager.js"></script>
-            <script type="text/javascript" src="/web_pwa_oca/static/src/js/webclient.js"></script>
+            <t t-if="not website">
```

Comment:
This is not really necessary... "web.assets_backend" is called in "web.webclient_bootstrap" that is rendered when loads "/web".

Reply:
Since web.webclient_bootstrap is used as a generic loader, I think it would be best to have all JS in assets_backend as it should be.
In that case someone can reroute JS below the fold if needed for backend.

Reply:
IIUC what @Tardo means is that `assets_backend` implies that you're already in the backend, so no need to check for website existence.

See my comment above too.